p4include/v1model.p4

```diff
@@ -582,17 +582,12 @@ extern void resubmit<T>(in T data);
  * data from the last such call is preserved.  See the v1model
  * architecture documentation (Note 1) for more details.
  */
-extern void recirculate<T>(in T data);
-
-/***
- * clone is in most ways identical to the clone3 operation, with the
- * only difference being that it never preserves any user-defined
- * metadata fields with the cloned packet.  It is equivalent to
- * calling clone3 with the same type and session parameter values,
- * with empty data.
- */
+extern void recirculate_field_list(bit<8> index);
 extern void clone(in CloneType type, in bit<32> session);
```

Comment:
There is another declaration of `extern void clone(in CloneType type, in bit<32> session)` in line 537. This causes compilation errors: 

```
 [--Werror=duplicate] error: clone: multiple matching declarations
...
v1model.p4(537): [--Werror=duplicate] error: Candidate: clone
extern void clone(in CloneType type, in bit<32> session);
            ^^^^^
v1model.p4(586): [--Werror=duplicate] error: Candidate: clone
extern void clone(in CloneType type, in bit<32> session);
            ^^^^^
```

Reply:
how come our tests pass?

Reply:
Not sure what flags/options we are using for the compilation (maybe @smolkaj can shed some light?). Anyway that declaration is indeed duplicated in v1model.p4 in this pull request (lines 537 and 586).

Reply:
I guess if you never call the method you don't get the error.
I will remove one of the two declarations, in the meantime can you please do the same if you want to try this?

Reply:
That makes sense, also maybe somebody should add a test for `clone` to p4c's test suite to avoid future regressions. Yeah I commented out that line and the error goes away.